packages/keycloak-user-management/src/forms/index.tsx

```diff
@@ -0,0 +1,284 @@
+import { ErrorMessage, Field, Formik } from 'formik';
```

Comment:
This component's tests throw an error:

```
  console.error
    Warning: Each child in a list should have a unique "key" prop.
    
    Check the render method of `Formik`. See https://fb.me/react-warning-keys for more information.
        in Option (created by Formik)
        in Formik (created by UserForm)
        in div (created by UserForm)
        in UserForm (created by WrapperComponent)
        in WrapperComponent
```

Reply:
addressed via https://github.com/OpenSRP/web/pull/39/commits/9afd8269acc9cc81d2c51c701f003e3a30c82283